### PR TITLE
adding the option functionality 

### PR DESCRIPTION
**Pull request by almutnaw**
So I created an option for the user to choose in html and did some javascript to decide which option have been chosen. Also I finished the regular card selection and all I have left now is to connect the Spanish deck.

### CODE REVIEW

**Comment by anisimon** [comment]
All checks failed but it can still merge? That seems strange.

**Comment by almutnaw** [comment]
Yeah, I don't know what's wrong with it. does anyone know?

File: dependency-reduced-pom.xml [hunk 1]
```diff
@@ -1,177 +1,177 @@
...
-        </exclusion>
-      </exclusions>
-    </dependency>
-  </dependencies>
-  <properties>
-    <ninja.version>5.2.2</ninja.version>
-    <jetty.version>9.2.10.v20150310</jetty.version>
-  </properties>
-</project>
-
```
Comment (paublo96) [inline dependency-reduced-pom.xml:177]: That's weird that this file was changed, but all of the changes are the same? I don't think it should affect anything though.

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 2]
```diff
@@ -200,6 +207,18 @@ <h1 style="color:blue;" id = "score">Score: 0</h1>
...
+    var x = document.getElementById("gameType").value;
+    console.log(x);
+    if(x == "Spanish"){
+        console.log("clicked spanish!!");
+    }
+    else{
+    console.log("clicked Normal!!");
+    $("#newGame").click();
+    }
+}
```
Comment (paublo96) [inline src/main/java/views/AcesUp/AcesUp.flt.html:221]: I think all you have to do to make this functional is create a new route calling the other deck type, then just call that route from the button.

**Comment by paublo96** [comment]
This looks good. We can figure out hooking up the button when we meet.

**Pull request merged**